### PR TITLE
[fronius] fix powerflowchannelpakku documentation

### PR DESCRIPTION
**Pull request by marcquark**
fixes openhab#15074

### CODE REVIEW

**Comment by lsiepel** [comment]
> fixes #15074

Don't know what is good or wrong, but it is not making it more clear.  Most systems show a + when charging/increasing and a - when discharging/decreasing. 

The label is about the battery, so from the battery perspective it is charging / increasing, so i would say a + is the best option.

**Comment by marcquark** [comment]
Agree that it would be more intuitive to have a positive number (+) when charging and a negative number (-) when discharging. However the inverter's API produces the opposite. So right now the documentation is wrong.

The other option is to multiply the value by -1 to make the documentation correct, but not sure if that's a smart thing to do considering it only affects one particular property and all the others do not alter the API output.

**Comment by lsiepel** [comment]
> Agree that it would be more intuitive to have a positive number (+) when charging and a negative number (-) when discharging. However the inverter's API produces the opposite. So right now the documentation is wrong.
> 
> The other option is to multiply the value by -1 to make the documentation correct, but not sure if that's a smart thing to do considering it only affects one particular property and all the others do not alter the API output.

Might be counter intuitive, but agree better to follow the API.

**Review by jimtng — APPROVED** [review]

File: bundles/org.openhab.binding.fronius/README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ The binding has no configuration options, all configuration is done at `bridge`,
 | `inverterdatadevicestatusstatuscode` | Number                   | Device status code<br />`0` - `6` Startup<br />`7` Running <br />`8` Standby<br />`9` Bootloading<br />`10` Error |
 | `powerflowchannelpgrid`              | Number:Power             | Grid Power (+ from grid, - to grid)                                                                               |
 | `powerflowchannelpload`              | Number:Power             | Load Power (+ generator, - consumer)                                                                              |
-| `powerflowchannelpakku`              | Number:Power             | Battery Power (+ charge, - discharge)                                                                             |
+| `powerflowchannelpakku`              | Number:Power             | Battery Power (- charge, + discharge)                                                                             |
```
Comment (jimtng) [inline bundles/org.openhab.binding.fronius/README.md:84]:
```suggestion
| `powerflowchannelpakku`              | Number:Power             | Battery Power (+ discharge, - charge)                                                                             |
```

Comment (jimtng) [reply bundles/org.openhab.binding.fronius/README.md:84]: My suggested change is just so it's more consistent, i.e. + first followed by - just like the others.

If you agree, you could either use github's feature to apply the suggestion or make the change on your copy and push an update to the branch.

In any case I'm not a main reviewer with merge rights so my suggested change is also subject to a review.

**Review by jimtng — APPROVED** [review]

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
This will make the documentation in sync with channel description:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml#L283

**Pull request merged**